### PR TITLE
03 hand interaction

### PR DESCRIPTION
**Pull request by aprilgittens**
Still adding text for each question in the project. I'm unsure if there were particular guidelines to follow for images? For now, I just included images without any sort of boxes or indicators to show what the user should click, etc. I can go back and add those once guidance is provided.

Also unsure if my images are embedded properly so I'm testing with one project first and then adding the rest of the images for the others.

Update: I figured it out for images :)

### CODE REVIEW

**Comment by aprilgittens** [comment]
For some reason, the images for the 'How to Add Manipulation Handler to Your Object' file are not rendering. Any clue why?

File: lessons/lesson-3/project/how-to-add-audio-feedback.md [hunk 1]
```diff
@@ -1,2 +1,45 @@
...
+
+Drag the object to receive the event and define **AudioSource.PlayOneShot** as the action to be triggered. **PlayOneShot** will play the audio clip.
+
+![](.Images/how_to_add_audio_feedback/add_object_and_action.PNG)
+
+![Add object and action](../../../.gitbook/assets/how_to_add_audio_feedback/object_action.PNG)
+
+Next, assign an audio clip to the trigger. You can find audio clips provided by MRTK by navigating to Assets > MixedRealityToolkit.SDK > StandardAssets > Audio. Once you've found a suitable audio clip, assign the audio clip to the **Audio Clip** field.
+
+[insert picture]
```
Comment (Yonet) [inline lessons/lesson-3/project/how-to-add-audio-feedback.md]: Looks great, Will you add the image here?

File: lessons/lesson-3/project/how-to-add-audio-feedback.md [hunk 2]
```diff
@@ -1,2 +1,45 @@
...
+
+You can now test the touch interaction using the in-editor simulation. Press the **Play** button to enter Game mode. Once in Game mode, hold the spacebar to bring up the hand and use the mouse to touch the object and trigger the sound effect.
+
```
Comment (Yonet) [inline lessons/lesson-3/project/how-to-add-audio-feedback.md:44]: Can you add the example link here as well? Thanks.

File: lessons/lesson-3/project/how-to-grab-and-move-an-object.md [hunk 3]
```diff
@@ -1,4 +1,22 @@
...
+You can move an object using one or two hands. This setting is dependent on the **Manipulation Type** parameter. The **Manipulation Type** can be limited to either:
+- One Handed Only
+- Two Handed Only
+- One and Two Handed
+
+Select the preferred **Manipulation Type** so that the user is restricted to use one of the available manipulation types.
+
+![Manipulation Type](../../../.gitbook/assets/how_to_grab_and_move_an_object/manipulation_type.PNG)
+
+note: is there a way to limit just to moving and not move scale or move rotate?
```
Comment (Yonet) [inline lessons/lesson-3/project/how-to-grab-and-move-an-object.md]: I see that you added the limit to axis, plz delete note. Thanks!

Comment (Yonet) [reply lessons/lesson-3/project/how-to-grab-and-move-an-object.md]: Please add a link to example scene.

File: lessons/lesson-3/project/how-to-style-bounding-box.md [hunk 4]
```diff
@@ -1,2 +1,63 @@
...
+
+You can change the color of the handles when an object is grabbed by assigning a material to the **Handle Grabbed Material** property.
+
+In the **Handles** section, drag a new material to the **Handle Grabbed Material** parameter.
+
+![Change Handle Grabbed material](../../../.gitbook/assets/how_to_style_bounding_box/handle_grabbed_parameter.PNG)
+
+When you press play, grab oen of the handles of the bounding box. The color of the handle will change to the color you dragged into the **Handle Grabbed Material** parameter.
+
+![New Handle Grabbed material](../../../.gitbook/assets/how_to_style_bounding_box/handle_grabbed_color.PNG)
```
Comment (Yonet) [inline lessons/lesson-3/project/how-to-style-bounding-box.md]: This section looks good but the end product(Cube with styled manipulation handlers) doesn't fit the mrtk style in the first picture. Can you make sure it does. Thanks!

**Review by Yonet — COMMENTED** [review]
Looking good! Please look at the comments to finish up.
Manipulation handler image links is not correct. You can check it by ctrl-clicking on the image link.

**Comment by aprilgittens** [comment]
@Yonet thanks for reviewing and providing feedback! i'll take a look and update in the AM. I left a few questions for you over in Teams as I'm currently blocked on completing the remaining stuff for Projects.

File: lessons/lesson-3/project/how-to-add-audio-feedback.md (hunk 1 above)
Comment (aprilgittens) [reply lessons/lesson-3/project/how-to-add-audio-feedback.md]: @Yonet I left a message for you in Teams regarding this - however, when using the branch from the seed project, I can't drag audio clips into the field. Unsure why it's not working? Are you able to drag audio from the MixedRealityToolkit.SDK folder to an Audio Source component event?

**Comment by aprilgittens** [comment]
@Yonet just added 'how to add button prefabs'

File: lessons/lesson-3/project/how-to-add-audio-feedback.md [hunk 5]
```diff
@@ -1,2 +1,45 @@
-# How to add audio feedback?
```
Comment (Yonet) [inline lessons/lesson-3/project/how-to-add-audio-feedback.md:1]: @aprilspeight please keep the format and don't capitalize. The reason is, some titles are long like: "What do I need to do to make sure Android, iOS and HoloLens are using the same point as my anchor?". It doesn't look good when all capital. Thanks!

**Comment by aprilgittens** [comment]
@Yonet I just finished the remaining updates. They're in the last 2 commits.

**Pull request merged**